src/node/fetch/fetch.node.test.ts

```diff
@@ -390,34 +374,44 @@ describe('NodeFetch', () => {
         });
       });
     });
-  });
-
-  describe('prepareHeadersInit', () => {
-    let nodeFetch: NodeFetch;
 
-    type OverrideNodeFetch = {
-      prepareHeadersInit: (data: any) => any | undefined;
-    };
+    // this test works under the assumption that node-fetch returns multi-valued headers as arrays
+    // this is not true for node-fetch 2.x
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip('should correctly send and receive multi-valued headers', async () => {
```

Comment:
question: I tried to run the test and it actually passed, even though the node-fetch version is node-fetch 2.6.7. Why is the test skipped?

Reply:
Yeah, it is passing because we are mocking `fetch` and mocked function returns header values as an array as it should. I just skipped it, but I will also change it to use fetch instance, so it is not confusing.

Reply:
Thanks for explanation. The mock was actually behaving according to node-fetch v3.

Reply:
Exactly mock was working as it would be result from node-fetch 3.0 instead of actually used 2.0.